wremnants/include/utils.h

```diff
@@ -32,6 +32,18 @@ constexpr double muon_mass = 0.1056583745;
     //     std::cout << var << std::endl;
     //     return 1;
     // }
+
+float pt_2(float pt1, float phi1, float pt2, float phi2) {
```

Comment:
There is another syntax for the lowPU (https://github.com/WMass/WRemnants/blob/main/scripts/histmakers/mw_lowPU.py#L278-L280). I don't know which syntax is better but it would be nice to have the same in both cases.

Reply:
I guess it depends on the availability of the x-y components for that formula, I think the one I am using it more readable, it is also similar to the mt one, where you pass the muon and met pt-phi. But I can change it if there is a strong preference

Reply:
To use a C++ function makes sense to me, can you change this in the lowPU script, also removing the definition of "pxll" and "pyll" then?

Reply:
Checking again, I would actually suggest changing the low PU one to match the other, since in the line you link there are three defines but two are only used in the third one, it seems a waste of CPU time.

Reply:
(sorry, I wrote this before I refreshed the page, so before seeing your comment, I will do it)